### ISSUE TITLE
My domain is throwing an error in the nu validator

### ISSUE THREAD

**Issue by mikemai2awesome** [validator/validator#2030]
### Spec URL

https://validator.w3.org/nu/

### Test case

```html
1. Navigate to https://validator.w3.org/nu/
2. Input https://mikemai.net
3. Click the check button
4. Receive an error message of "IO Error: HTTP resource not retrievable. The HTTP status from the remote server was: 403. https://mikemai.net/"
```

**Comment by mikemai2awesome** [comment]
hi there, i am not sure if i am doing something wrong with my website or if the validator indeed has a bug. i would like to know what is triggering this error.

**Comment by 1jj** [comment]
You didn't make an error. It fails at https://validator.w3.org/ also, which is (almost) the same service.

My assumption is:
Validator's  IP addresses are blocked by your server or hosting provider to reduce robot traffic.  
I had and have such issues with a specific low-cost provider.

EDIT:
You can paste your HTML code into the text input field (use "Check by" select)

**Comment by mikemai2awesome** [comment]
@1jj  thank you for the tip. i will check my cloudflare setting, maybe there's something there i am not aware of.

**Comment by sideshowbarker** [comment]
How @mikemai2awesome — as 1jj indicated at https://github.com/validator/validator/issues/2030#issuecomment-3756996763, it seems something in the pipeline from your end is blocking requests from the checker’s IP address/range.

And also 1jj also pointed out: the workaround is to instead send the HTML of your document to the checker: Either by using the checker’s _“Check by: text input”_ option or else by using the _“Check by: file upload”_ option.

But there’s otherwise nothing we can do in the checker code to prevent the blocking you’re running into. So I’m going ahead and closing this. But that’s not a meant a signal to avoid commenting here further — feel free to continue to comment here if you want. Closing the issue just indicates we don’t anticipate needing to take any action needed — unless there’s some new information that comes up which suggests we could make a code change here to mitigate the problem (or something).
